docs/index.rst

```diff
@@ -183,6 +197,10 @@ method, where the block id needs to be supplied and can be accessed from the
 Error handling
 ==============
 
+The success of the last call to `get_blockid`, `get_positions`, `get_box` and
+related methods can be checked using the :class:`TNGCurrentIntegratorStep.read_success`
+property, which returns `True` if the attempt to read was successful or `False` if not.
```

Comment:
Explain what "successful" means. In particular, can it fail because (1) EOF, (2) file corruption, (3) no data stored for the step? Are these all equally unsuccessful?

Comment on reST: use double-backticks to format as code. Single backtick indicates a role in reST/sphinx (such as 
```reST
This indicates a class :class:`Universe` with the ":class:" role.
If I write `Universe` then it assumes the default role (whatever that is).
To explicitly mark up code use double back-ticks, e.g., ``u = mda.Universe(...)``, or ``(True != False) == True`` is ``True``.
```

and it just uses the default role

Reply:
sorry for delay, will get to this asap.